libs/styles/styles.css

```
@@ -1,134 +1,586 @@
-/*** Variables ***/
+/* stylelint-disable comment-empty-line-before */
+/* stylelint-disable custom-property-empty-line-before */
 
- :root {
+:root {
   --global-height-nav: 64px;
   --global-height-breadcrumbs: 33px;
   /* stylelint-disable-next-line custom-property-pattern */
   --global-height-navPromo: 65px;
   --feds-totalheight-nav: calc(var(--feds-height-nav, --global-height-nav) + var(--feds-height-breadcrumbs, --global-height-breadcrumbs));
 
-  /* Colors */
-  --link-color: #035fe6;
-  --link-hover-color: #136ff6;
+  /* Fonts */
+  --body-font-family: 'Adobe Clean', adobe-clean, 'Trebuchet MS', sans-serif;
+  --headline-font-family: 'Adobe Headline', adobe-headline, 'Adobe Clean', sans-serif; /* new */
```

We should probably mimic the value from `--body-font-family` and just prefix it with the headline variants
```suggestion
  --headline-font-family: 'Adobe Headline', adobe-headline, 'Adobe Clean', adobe-clean, 'Trebuchet MS', sans-serif; /* new */
```